index.html

```diff
@@ -2,94 +2,113 @@
-				</section>
-				<section>
-					<h2>Friends of Oz</h2>
-					<div class="main_friends">
-						<div class="friend">
-							<h3>Dorothy</h3>
-							<div>
-								<img src="images/dorothy.jpg" alt="Dorothy">
-							</div>
-						</div>
-						<div class="friend">
-							<h3>Lion</h3>
-							<div>
-								<img src="images/lion.jpg" alt="Lion">
-							</div>
-						</div>
-						<div class="friend">
-							<h3>Scarecrow</h3>
-							<div>
-								<img src="images/scarecrow.jpg" alt="Scarecrow">
-							</div>
-						</div>
-						<div class="friend">
-							<h3>Tin Man</h3>
-							<div>
-								<img src="images/tinman.jpg" alt="Tin Man">
-							</div>
-						</div>
-					</div>
-				</section>
-			</div>
+        <main>
+            <div class="main_section">
+                <section>
+                    <div class="hero"><img src="images/oz.jpg"></div>
+                    <h1>The Wizard of Oz is a 1939 American musical fantasy film</h1>
+                    <p>When a tornado rips through Kansas, Dorothy (Judy Garland) and her dog, Toto, are whisked away in
+                        their house to the magical land of Oz. They follow the Yellow Brick Road toward the Emerald City
+                        to meet the Wizard, and en route they meet a Scarecrow (Ray Bolger) that needs a brain, a Tin
+                        Man (Jack Haley) missing a heart, and a Cowardly Lion (Bert Lahr) who wants courage. The wizard
+                        asks the group to bring him the broom of the Wicked Witch of the West (Margaret Hamilton) to
+                        earn his help.</p>
+                </section>
+                <section>
+                    <h2>Friends of Oz</h2>
+                    <div class="main_friends">
+                        <div class="friend">
+                            <h3>Dorothy</h3>
+                            <div>
+                                <img src="images/dorothy.jpg" alt="Dorothy">
+                            </div>
+                        </div>
+                        <div class="friend">
+                            <h3>Lion</h3>
+                            <div>
+                                <img src="images/lion.jpg" alt="Lion">
+                            </div>
+                        </div>
+                        <div class="friend">
+                            <h3>Scarecrow</h3>
+                            <div>
+                                <img src="images/scarecrow.jpg" alt="Scarecrow">
+                            </div>
+                        </div>
+                        <div class="friend">
+                            <h3>Tin Man</h3>
+                            <div>
+                                <img src="images/tinman.jpg" alt="Tin Man">
+                            </div>
+                        </div>
+                        <div class="friend">
+                            <h3>Toto</h3>
```

Comment:
Hey! Please make this "Toto The Dog" as we agreed! Thanks so much!

Reply:
Agreed